File: app/src/main/java/be/kdg/mobile_client/model/Room.java
package be.kdg.mobile_client.model;

import java.util.List;

import lombok.Data;

/**
 * A room that was retrieved by the game-service.
 */
@Data
public class Room {
    private int id;
    private String name;
    private GameRules gameRules;
    private List<Player> playersInRoom;
}

File: web-client/src/app/model/room.ts
import {GameRules} from './gamerules';
import {Player} from './player';

export class Room {
  id: number;
  name: string;
  gameRules: GameRules;
  playersInRoom: Player[];

  static create() {
    return {
      id: 0,
      name: '',
      gameRules: GameRules.create(),
      playersInRoom: [Player.create()]
    };
  }
}
